build.js

```diff
@@ -9,26 +10,36 @@ const stringifyObject = require('stringify-object');
 
 const pkg = require('./package.json');
 
-let array = Object.keys(require('spdx-license-list'));
-let files = [
-  {
-    path: pkg.main,
-    contents: JSON.stringify(array, null, '  ')
-  },
-  {
-    path: require('./bower.json').main,
-    contents: `window.spdxLicenseIds = ${stringifyObject(array, {indent: '  '})};`
-  }
-];
+http.get('http://spdx.org/licenses/licenses.json', response => {
+  let buffer = '';
+  response
+    .on('data', data => {
+      buffer += data;
+    })
+    .on('end', () => {
+      const json = JSON.parse(buffer);
```

Comment:
I hear [it's even easier in Ruby](https://github.com/kemitchell/rubygems/commit/a2630898bdb9cfab4d3ef6965112644276639740#diff-cbc4f9532c53d8e89750066d832033bdR4)! :smirk: